### PR TITLE
ZHASH, PHI2, PHI1612, ...

### PR DESCRIPTION
**Pull request by papampi**
ZHASH algo and Z_EWBF

### CODE REVIEW

**Comment by papampi** [comment]
Better ideas for ewbf zhash miner?

File: 1bash.template [hunk 1]
```diff
@@ -591,10 +605,10 @@ BTCZ_PORT="5586"
 BTCZ_WORKER="$WORKERNAME"
 
 # Bitcoin Gold (BTG)
-BTG_ADDRESS="replace_with_your_address"
-BTG_EXTENSION_ARGUMENTS=""
-BTG_POOL="replace_with_your_BTG_POOL"
-BTG_PORT="replace_with_your_BTG_PORT"
+BTG_ADDRESS="$MPH_ADDRESS"
```
Comment (LuKePicci) [inline 1bash.template]: should be "replace_with_your_address" to be consistent with all the other coins here which also have mph as pool example

**Comment by LuKePicci** [comment]
Update the miners submodule moving ewbf to the new path and separate installers, then push the updated reference here and we can merge.

**Comment by LuKePicci** [comment]
if the miners are ready commit the submodule reference update

**Review by LuKePicci — APPROVED** [review]

**Pull request merged**